e2e/scenario_test.go

```diff
@@ -435,32 +410,6 @@ func Test_marinerv2CustomSysctls(t *testing.T) {
 	})
 }
 
-// Returns config for the 'gpu' E2E scenario
-func Test_marinerv2gpu(t *testing.T) {
-	RunScenario(t, &Scenario{
-		Description: "Tests that a GPU-enabled node using a MarinerV2 VHD can be properly bootstrapped",
-		Tags: Tags{
-			GPU: true,
-		},
-		Config: Config{
-			Cluster: ClusterKubenet,
-			VHD:     config.VHDCBLMarinerV2Gen2,
-			BootstrapConfigMutator: func(nbc *datamodel.NodeBootstrappingConfiguration) {
-				nbc.ContainerService.Properties.AgentPoolProfiles[0].VMSize = "Standard_NC6s_v3"
-				nbc.ContainerService.Properties.AgentPoolProfiles[0].Distro = "aks-cblmariner-v2-gen2"
-				nbc.AgentPoolProfile.VMSize = "Standard_NC6s_v3"
-				nbc.AgentPoolProfile.Distro = "aks-cblmariner-v2-gen2"
-				nbc.ConfigGPUDriverIfNeeded = true
-				nbc.EnableGPUDevicePluginIfNeeded = false
-				nbc.EnableNvidia = true
-			},
-			VMConfigMutator: func(vmss *armcompute.VirtualMachineScaleSet) {
-				vmss.SKU.Name = to.Ptr("Standard_NC6s_v3")
-			},
-		},
-	})
-}
-
 func Test_marinerv2gpu_azurecni(t *testing.T) {
```

Comment:
maybe we should keep azure_cni + gpu on the latest AzureLinux ? marinerv2 is somewhat deprecated now no ?

Reply:
is it deprecated?